package.json

```diff
@@ -62,18 +65,18 @@
     "superagent": "^6.1.0",
     "supertest": "^6.1.6",
     "ts-jest": "^27.1.1",
+    "tsup": "^6.5.0",
     "typescript": "4.4.4",
-    "wait-for-expect": "^3.0.2"
+    "wait-for-expect": "^3.0.2",
+    "web-encoding": "^1.1.5"
   },
   "dependencies": {
     "@open-draft/until": "^1.0.3",
     "@remix-run/web-fetch": "^4.3.1",
```

Comment:
We need to ensure that we bundle this Fetch polyfill from Remix. Should we move it to dev dependencies as well?

Reply:
Done